crates/topos-clock/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+//! This crate is responsible for managing the clock pace.
+//!
+//! The Clock is responsible of giving informations about Epoch and Delta timing by exposing
```

Comment:
```suggestion
//! The Clock is responsible of giving informations about Epoch and Delta timings by exposing
```

Is "delta' the same as "block time"?

Reply:
No, `Delta` are an amount of blocks. The amount of blocks isn't specified for now.

Reply:
got it, thanks